trace-utils/src/send_data.rs

```diff
@@ -725,16 +737,27 @@ mod tests {
     async fn request_msgpack() {
         let server = MockServer::start_async().await;
 
+        let header_tags = HEADER_TAGS;
         let mock = server
             .mock_async(|when, then| {
                 when.method(POST)
+                    .header(HEADER_DD_TRACE_COUNT, "1")
```

Comment:
`when` asserts the header is present, right? Not just setting up the mock, right?

Reply:
correct, if the headers aren't set correctly the mock won't return a 200 and the test will fail.